backend/typescript/graphql/types/residentType.ts

```diff
@@ -0,0 +1,57 @@
+import { gql } from "apollo-server-express";
+
+// TODO: Look into custom types for dates and date time types
+
+const residentType = gql`
+  type ResidentDTO {
+    id: ID!
+    first_name: String!
```

Comment:
With the new camel case changes for the DTO's and the mappers, this also has to be changed to camel case otherwise the postman queries wont work (same for all the DTO's on this file)